src/main/kotlin/widgets/Selector.kt

```diff
@@ -0,0 +1,178 @@
+package widgets
+
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.subjects.PublishSubject
+import javafx.application.Platform
+import javafx.beans.property.SimpleObjectProperty
+import javafx.geometry.Insets
+import javafx.geometry.Pos
+import javafx.scene.control.ComboBox
+import javafx.scene.layout.Priority
+import javafx.scene.paint.Color
+import tornadofx.*
+import java.util.*
+
+/**
+ * This class is used to make allow a user to select items from a filterable list
+ * and select which of those that are selected as the preferred, or default, item.
+ */
+
+class Selector(selectionData : List<DataSelectionInterface>,
```

Comment:
What selector?

Reply:
Changed to DataListSelector, although I can change it to any more inspired name

Reply:
Also changed the stylesheet name appropriately